packages/polindrome/presentation/lib/state_man/base_bloc.dart

```diff
@@ -0,0 +1,42 @@
+
+import 'dart:async';
+import 'package:flutter/widgets.dart';
+import 'package:presentation/state_man/data/bloc_state.dart';
+
+abstract class BaseBloc<A>{
+  Stream<BlocData?> get getStream;
+  void initState();
+  void dispose();
+}
+class BlocImpl<A> implements BaseBloc<A>{
+  final _data = StreamController<BlocData<A?>>();
+  final _blocData = BlocData.init();
+
+  @override
+  void dispose(){
+    _data.close();
+  }
+  @override
+  void initState(){
+  }
+
+  @override
+  Stream<BlocData?> get getStream => _data.stream;
+
+  @protected
+    void handleData({
+      bool? isLoading,
+      bool? isValid,
```

Comment:
Do you use 'isValid' anywhere ?  
I reviewed the code and didn't see the use of this variable.
Maybe it should be removed ?